envoy/tests/docker/default/Dockerfile-frontenvoy

```diff
@@ -0,0 +1,5 @@
+ARG ENVOY_VERSION
+FROM envoyproxy/envoy:${ENVOY_VERSION}
+
+RUN apt-get update && apt-get -q install -y curl
+CMD /usr/local/bin/envoy -c /etc/front-envoy.yaml --service-cluster front-proxy
```

Comment:
What is this doing that the official image from the docker hub can't do?

Reply:
This is the only way to load the configuration https://www.envoyproxy.io/docs/envoy/latest/start/start#using-the-envoy-docker-image

Reply:
The link doesn't help honestly. Why are you installing curl? And why can't you just mount wherever the config file is to `/etc/envoy/envoy.yaml`?